src/helpers/build_specific_object/index.js

```diff
@@ -28,27 +29,29 @@ function addSpecificItemsToSet(set: Set<string>, items: Specific): Set<string> {
  */
 export default function buildSpecificObject(specific: Specific = []): SpecificObject {
 
-    const result: SpecificObject = {
-        only: new Set(),
-        not: new Set()
-    };
+    const result: SpecificObject = {};
 
     if (!specific) { return result; }
 
     if (
         typeof specific === 'string'
         || Array.isArray(specific)
     ) {
-        addSpecificItemsToSet(result.only, specific);
+        if (specific.length === 0) { return result; }
+
+        result.only = {};
+        addSpecificItemsToGroup(result.only, specific);
         return result;
     }
 
     if (specific.only) {
-        addSpecificItemsToSet(result.only, specific.only);
+        result.only = {};
+        addSpecificItemsToGroup(result.only, specific.only);
     }
 
     if (specific.not) {
-        addSpecificItemsToSet(result.not, specific.not);
+        result.not = {};
+        addSpecificItemsToGroup(result.not, specific.not);
```

Comment:
I propose instead of `Passable.js` & `passable_args.js` & `build_specific_object.js`:

### Passable.js
```js
// @flow

import testRunner from './test_runner';
import PassableResult from './result_object';
import PassableSpecific from './specific';
import { runtimeError } from 'Helpers';
import { Errors } from 'Constants';

class Passable {

    specific: SpecificObject;
    result: ResultObject;
    test: TestProvider;

    constructor(name: string, specific: Specific, tests: TestsWrapper) {
        if (!arguments.length) {
            throw runtimeError(Errors.PASSABLE_ARGS_NO_ARGS);
        }

        if (typeof name !== 'string') {
            throw runtimeError(Errors.INVALID_FORM_NAME, typeof name);
        }

        if (typeof tests !== 'function') {
            throw runtimeError(Errors.PASSABLE_ARGS_UNEXPECTED_ARGS_1, typeof tests);
        }

        this.specific = new PassableSpecific(specific);
        this.result = new PassableResult(name);

        return tests(this.test);
    }

    test = (fieldName: string, statement: string, test: TestFn, severity: Severity) => {
        if (this.specific.excludes(fieldname)) {
            this.result.addToSkipped(fieldName);
            return this.result;
        }

        this.result.initFieldCounters(fieldName);

        if (typeof test !== 'function') {
            return this.result;
        }

        const isValid: boolean = testRunner(test);

        if (!isValid) {
            this.result.fail(fieldName, statement, severity);
        }

        return this.result.bumpTestCounter(fieldName);
    }
}

export default Passable;
```

### PassableSpecific.js
```js
class PassableSpecific {
    constructor(specific) {
        this.only = {};
        this.not = {};

        if (!specific) { return; }

        if (typeof specific === 'string' || Array.isArray(specific)) {
            if (specific.length === 0) { return; }
            this.populateGroup(this.only, specific);
            return;
        }

        if (specific.only) {
            this.populateGroup(this.only, specific.only);
        }

        if (specific.not) {
            this.populateGroup(this.not, specific.not);
        }
    }

    populateGroup(group, specific) {
        Array.prototype.concat(specific).forEach(
            (field) => group[field] = true
        );
    }

    excludes(name) {
        return !this.only[fieldName] || this.not[name];
    }
}
```

WDYT?